scanner/src/funTest/assets/scanner-integration-all-pkgs-expected-ort-result.yml

```diff
@@ -406,7 +406,57 @@ scanner:
           path: "pkg4/pkg4.txt"
           start_line: -1
           end_line: -1
-  files: []
+  files:
```

Comment:
Commit message: Something's wrong with the backticks around "sha1-sum", the opening one seems to be the wrong way. Why is is called `sha1-sum` instead of `sha1`, BTW?